_episodes/09-pull-requests.md

```diff
@@ -155,28 +234,48 @@ Writing objects: 100% (4/4), 783 bytes | 0 bytes/s, done.
 Total 4 (delta 3), reused 0 (delta 0)
 remote: Resolving deltas: 100% (3/3), completed with 3 local objects.
 To https://github.com/USERNAME/countries.git
-   2037539..79a312a  master -> master
+   2037539..79a312a  addFrance -> addFrance
 ~~~
 {: .output}
 
 Next go to your forked github version of the repo and reload the page.
```

Comment:
I didn't notice this image the first time through. I think it's super important to have them go through this step and see the new branch on GitHub. Nice!